crates/forge/src/test_filter.rs

```diff
@@ -163,6 +163,7 @@ mod tests {
     fn filtering_tests() {
         let mocked_tests = TestTargetWithResolvedConfig {
             sierra_program: program_for_testing(),
+            sierra_program_path: Default::default(),
```

Comment:
Nit: I think it would be clearer to use `Arc::default()` so the reader knows exactly what type it is. The same applies to other similar occurrences.  
If you want to commit to this approach, you can disallow the `clippy::default_trait_access` lint, and Clippy will automatically highlight these places for you. (Should probably do this in separate pr)